component/custom/custom_loader.go

```diff
@@ -0,0 +1,67 @@
+package custom
+
+import (
+	"os"
+	"path/filepath"
+	"plugin"
+	"strings"
+
+	"github.com/Azure/azure-storage-fuse/v2/common/log"
+	"github.com/Azure/azure-storage-fuse/v2/external"
+	"github.com/Azure/azure-storage-fuse/v2/internal"
+)
+
+func init() {
+	// Environment variable for directory containing the plugin .so files.
+	pluginDirPath := os.Getenv("BLOBFUSE_PLUGIN_DIRECTORY")
```

Comment:
Are we keeping this only in Env variables or in the mount command this can be supplied as CLI parameter as well?

Reply:
Can you also add some documentation on it in README mentioning the env variables and other parameters if any.

Reply:
This is only in Env variables right now.